DataConnectors/CiscoUmbrella/CiscoUmbrella_API_FunctionApp.json

```diff
@@ -0,0 +1,176 @@
+{
+    "id": "CiscoUmbrellaDataConnector",
+    "title": "Cisco Umbrella (Preview)",
+    "publisher": "Cisco",
+    "descriptionMarkdown": "The Cisco Umbrella data connector provides the capability to ingest [Cisco Umbrella](https://docs.umbrella.com/) events stored in Amazon S3 into Azure Sentinel using the Amazon S3 REST API. Refer to [Cisco Umbrella log management documentation](https://docs.umbrella.com/deployment-umbrella/docs/log-management) for more information.",
+    "additionalRequirementBanner": "These queries and workbooks are dependent on a parser based on Kusto to work as expected. Follow the steps to use this Kusto functions alias **Cisco_Umbrella** in queries and workbooks. [Follow steps to get this Kusto functions>](https://raw.githubusercontent.com/Azure/Azure-Sentinel/CiscoUmbrellaConnector/Parsers/CiscoUmbrella/Cisco_Umbrella) ",
+    "graphQueries": [
+        {
+            "metricName": "Cisco Umbrella DNS Logs",
+            "legend": "DNS",
+            "baseQuery": "Cisco_Umbrella_dns_CL"
+        },
+        {
+            "metricName": "Cisco Umbrella Proxy Logs",
+            "legend": "Proxy",
+            "baseQuery": "Cisco_Umbrella_proxy_CL"
+        },
+        {
+            "metricName": "Cisco Umbrella IP Logs",
+            "legend": "IP",
+            "baseQuery": "Cisco_Umbrella_ip_CL"
+        },
+        {
+            "metricName": "Cisco Umbrella Cloud Firewall Logs",
+            "legend": "Cloud Firewall",
+            "baseQuery": "Cisco_Umbrella_cloudfirewall_CL"
+        }
+    ],
+    "sampleQueries": [
+        {
+            "description" : "All Cisco Umbrella Logs",
+            "query": "Cisco_Umbrella_func\n | sort by TimeGenerated desc"
+        },
+        {
```

Comment:
Since we are taking a dependency on the parser, is it possible to modify the 4 other sample queries to also leverage the   Cisco_Umbrella Kusto function? That way the results will have normalized query results.

Reply:
sample queries updated to use Cisco_Umbrella Kusto function